src/infra/repos/postgresql/event-repository.ts

```diff
@@ -1,8 +1,12 @@
-import { type AddEventRepository, type LoadEventByDateRepository } from '@/data/protocols'
+
+import {
+  type AddEventRepository,
+  type LoadTasksByDateRepository
+} from '@/data/protocols'
 import { prisma as db } from './prisma'
 import { type AddManyEventsRepository } from '@/data/protocols/db/event/add-many-events-repository'
 
-export class EventRepository implements AddEventRepository, LoadEventByDateRepository, AddManyEventsRepository {
+export class EventRepository implements AddEventRepository, AddManyEventsRepository {
```

Comment:
Faltou implementar o `LoadTasksByDateRepository`
```suggestion
export class EventRepository implements AddEventRepository, AddManyEventsRepository, LoadTasksByDateRepository {
```